shared/chat/conversation/info-panel/notifications/container.js

```diff
@@ -8,6 +8,81 @@ import {Notifications, type Props} from '.'
 import {connect, type TypedState} from '../../../../util/container'
 import {type DeviceType} from '../../../../constants/types/devices'
 
+type DispatchProps = {
+  _resetSaveState: (conversationIDKey: Types.ConversationIDKey) => void,
+  _onMuteConversation: (conversationIDKey: Types.ConversationIDKey, muted: boolean) => void,
+  _onSetNotification: (
+    conversationIDKey: Types.ConversationIDKey,
+    deviceType: DeviceType,
+    notifyType: Types.NotifyType
+  ) => void,
+  _onToggleChannelWide: (conversationIDKey: Types.ConversationIDKey) => void,
+}
+
+const mapDispatchToProps = (dispatch: Dispatch): DispatchProps => ({
+  _resetSaveState: (conversationIDKey: Types.ConversationIDKey) =>
+    dispatch(ChatGen.createSetNotificationSaveState({conversationIDKey, saveState: 'unsaved'})),
+  _onMuteConversation: (conversationIDKey: Types.ConversationIDKey, muted: boolean) => {
+    dispatch(ChatGen.createMuteConversation({conversationIDKey, muted}))
+  },
+  _onSetNotification: (
+    conversationIDKey: Types.ConversationIDKey,
+    deviceType: DeviceType,
+    notifyType: Types.NotifyType
+  ) => dispatch(ChatGen.createSetNotifications({conversationIDKey, deviceType, notifyType})),
+  _onToggleChannelWide: (conversationIDKey: Types.ConversationIDKey) =>
+    dispatch(ChatGen.createToggleChannelWideNotifications({conversationIDKey})),
+})
+
+type OwnProps = {
```

Comment:
All of these are derived from the convIDKey/inbox, so we pretty much have to pass them all down

Reply:
Why is this true. Do what you did before in mapstatetoprops again

Reply:
I guess I don't understand something. If I make another `mapStateToProps` here for all the inbox-derived props, what's the point of splitting up into two components?